src/css/docs-ndl.css

```
@@ -498,25 +504,20 @@ body.docs-ndl .banner .button {
 
 /* highlights */
 
-body.docs-ndl .highlights .sectionbody > div.openblock {
-  flex: 1 1 40%;
-}
-
-body.docs-ndl .highlights .sectionbody > div.icon {
-  flex: 0 1 52%;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
-  padding-left: 4rem;
-  padding-right: 5.438rem;
+body.docs-ndl .highlights .sectionbody > div {
+  flex: 0 1 50%;
+  justify-content: center;
   align-self: center;
+  padding: 1rem;
```

If you agree with my comment about using display:flow, then I think these paddings would improve the justification of the text:

```suggestion
  padding: 1rem 2rem 1rem;  
```

---

Let's go with this. I've also added center justification to the image.